ci/jenkins/bin/clang-analyzer.sh

```diff
@@ -27,6 +27,7 @@ NOCLEAN=${NOCLEAN:-}
 OUTPUT_BASE=${OUTPUT_BASE:-/home/jenkins/clang-analyzer}
 
 # Options
+#options="--status-bugs --keep-empty"
```

Comment:
Not clear what the point of this is.

Reply:
I have to restore that once we fix all the remaining Clang Analyzer bugs. Removing the --status-bugs is what allows us to not mark the builds as failures.